backend/.env.example

```
@@ -3,3 +3,4 @@ SEMANTIC_SCHOLAR_API_KEY=""
 ANTHROPIC_API_KEY=""
 SUPABASE_URL=""
 SUPABASE_SERVICE_ROLE_KEY=""
+ACTOR_KEY_SECRET=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)